Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py

```diff
@@ -1,7 +1,7 @@
-from typing import Union
+import urllib3
 from CommonServerPython import *
 # Disable insecure warnings
-requests.packages.urllib3.disable_warnings()
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
https://stackoverflow.com/questions/27981545/suppress-insecurerequestwarning-unverified-https-request-is-being-made-in-pytho#28002687